code/style.css

```diff
@@ -1,4 +1,173 @@
-html {
-  background: #0025ff;
-  color: #fff;
+@import url("https://fonts.googleapis.com/css2?family=Balsamiq+Sans&display=swap");
+
+body {
+  background-image: linear-gradient(
+    120deg,
+    rgb(232, 204, 232),
+    rgb(153, 63, 153)
+  );
+  color: black;
+  margin: 0;
+  font-family: "Balsamiq Sans", cursive;
+}
+
+/****Header*****/
+.header-container {
+  background-color: black;
+  color: rgb(232, 204, 232);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.top-title {
+  margin-left: 15px;
+}
+.logo {
+  display: none;
+  width: 80px;
+}
+
+.nav-container {
+  display: none;
+}
+
+.nav-link {
+  text-decoration: none;
+  color: rgb(232, 204, 232);
+  padding: 10px;
+}
+
+/*Big news section*/
+
+.big-news-container {
+  margin: 50px;
+  border-radius: 25px;
+  background-color: whitesmoke;
+  padding: 15px;
+  display: flex;
+  justify-content: center;
+}
+.big-news-title {
+  margin-left: 30px;
+}
+
+.big-news-title:hover {
+  color: rgb(232, 204, 232);
+}
+
+.big-news-items {
+  margin: 0px 15px 15px;
+  display: flex;
+  justify-content: space-evenly;
+}
```

Comment:
This could maybe use `flex-direction: column;` in mobile - it gets a little crowded in there otherwise 😇 Two images for comparison:

<img width="464" alt="Skärmavbild 2021-02-02 kl  15 19 29" src="https://user-images.githubusercontent.com/51529201/106613044-0cc35a00-656a-11eb-86a0-0b51be767540.png">
<img width="444" alt="Skärmavbild 2021-02-02 kl  15 19 42" src="https://user-images.githubusercontent.com/51529201/106613070-1482fe80-656a-11eb-9dcc-bdb3ec8c287a.png">